L/libjulia/common.jl

```diff
@@ -444,6 +449,12 @@ function build_julia(ARGS, version::VersionNumber; jllversion=version)
         push!(dependencies, Dependency(get_addable_spec("LibUnwind_jll", v"1.8.1+2"); platforms=filter(!Sys.isapple, platforms)))
         push!(dependencies, Dependency(get_addable_spec("LLVMLibUnwind_jll", v"19.1.4+0"); platforms=filter(Sys.isapple, platforms)))
         push!(dependencies, BuildDependency(get_addable_spec("LLVM_full_jll", v"18.1.7+3")))
+    elseif version.major == 1 && version.minor == 13
```

Comment:
Uhm, there's something wrong here, both 1.12 and 1.13 will use openssl in place of mbedtls (completely removed).  I guess #10423 was already wrong.

Reply:
Thanks for the catch! Does the updated handling now look right to you?

Reply:
At a quick glance yes but I haven't double checked the versions numbers.

Reply:
While I agree we should get this right, AFAICT this does not affect the produced libjulia because we don't like against OpenSSL or MBedTLS on the C/C++ level, only the stdlib does. Or am I mistaken there?

The same probably also holds for e.g. SuiteSparse_jll and a bunch more

If this is right, then *ideally* the Julia build system would allow us to build the kernel w/o downloading these JLLs first...